### ISSUE TITLE
Suggestion for the UTorrent App Module

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#743]
**Reported by elliott94 on 2010-07-04 07:41**
Currently, using the latest snapshot, NVDA is successfully able to obtain the Files List using display hooks. However, it doesn't announce to the user whether the focused file is checked or unchecked.
Using display hooks, could this information be obtained?

**Comment by leonardder** [comment]
@elliott94: Do you still consider this issue to be valid?

**Comment by k-kolev1985** [comment]
@leonardder, yes, the issue is still valid, even with the latest version of uTorrent. And it probably still can't be fixed, unfortunately. I doubt that the uTorrent team has done something to improve that.
@jcsteh, please do not use pixel color if possible, because that might not work for color themes in Windows different from the one used while creating the workaround on your machine.

**Comment by Adriani90** [comment]
This issue is still reproducible.

**Comment by Adriani90** [comment]
I have tested with UTorrent 3.5.3.

**Comment by Adriani90** [comment]
@elliott94 is this still reproducible in UTorrent 3.5.5 (last stable version) with NVDA 2019.3.1 or newer?

**Comment by Adriani90** [comment]
I can reproduce this in uTorrent 3.5.5 as follows:
1. Create a torrent file containing more than one file of any format
2. Open it in uTorrent
Actual: a list shows up where you can choose which file should be downloaded. However, NVDA does not report if the files are selected or not. Exploring with object navigation I can see that there is the patern "selected" or "not selected" as a child to the file name.

**Comment by Adriani90** [comment]
In utorrent 3.6.0 there is no selection patern in the child object visible, how ever you can exclude files from download by pressing right click on the file and choosing "exclude". I am closing this since the interface of utorrent changed over time. If you still can reproduce this, please open a new issue testing in the most current version of utorrent.
Note that you will need the java access bridge to be activate for this to work, I've installed both the 32bit and the 64bit java access bridge and enabled them via jabswitch -enable command in the powershell.
